Build-Scripts/gen-dependencies.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+HOMEBREW_BIN="${HOMEBREW_ROOT:-/opt/homebrew}/bin"
```

Comment:
Does modern homebrew installation set this variable? Mine (a few years old) definitely didn't.

Anyway why not use `brew config | grep "HOMEBREW_PREFIX:" |  awk -F ' '  '{print $2}'` or something similar to find it when env var is not set?